tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -61,34 +67,25 @@ fn everything() {
         &ledger_addr,
     ];
 
-    for &dry_run in &[true, false] {
-        let tx_args = if dry_run {
-            vec![tx_args.clone(), vec!["--dry-run"]].concat()
-        } else {
-            tx_args.clone()
-        };
-        let mut anomac_tx = run!(
-            test,
-            Bin::Client,
-            tx_args,
-            Some(CLIENT_COMMAND_TIMEOUT_SECONDS)
-        )
-        .unwrap();
+    let mut anomac_tx = run!(
+        test,
+        Bin::Client,
+        tx_args,
+        Some(CLIENT_COMMAND_TIMEOUT_SECONDS)
+    )
+    .unwrap();
 
-        if !dry_run {
-            if !cfg!(feature = "ABCI") {
-                anomac_tx.exp_string("Transaction accepted").unwrap();
-            }
-            anomac_tx.exp_string("Transaction applied").unwrap();
-        }
-        // TODO: we should check here explicitly with the ledger via a
-        //  Tendermint RPC call that the path `value/#EthBridge/queue`
-        //  is unchanged rather than relying solely  on looking at anomac
-        //  stdout.
-        anomac_tx.exp_string("Transaction is invalid").unwrap();
-        anomac_tx
-            .exp_string(&format!("Rejected: {}", ETH_BRIDGE_ADDRESS))
-            .unwrap();
-        anomac_tx.assert_success();
+    if !cfg!(feature = "ABCI") {
```

Comment:
We really need to catch the flake with logs of what exactly is going on in the ledger (and the client). A flaky error can happen at the first client assertion we make e.g.

```rust
anomac_tx.exp_string("Transaction accepted").unwrap();
```
```
Message:  called `Result::unwrap()` on an `Err` value: 
   0: Reached a timeout for expect type of command
       Needle: Transaction accepted
```

I think it's possible the ledger is shutting down for some reason before the client makes its call. As we wait for the ledger to have committed at least one block, Tendermint is presumably healthy for at least a bit, so not sure why this is happening (if it is).

Reply:
That's entirely possible. But catching it and logging it propoerly has been our white whale

Reply:
after https://github.com/anoma/anoma/pull/1202 merges in we will have full logs for all the `anoma*` commands we run, then we could try making the logs available from CI somehow. Maybe making the logs available for download, or just spewing them out to stdout when there is a failure